services/ledpixel.md

```diff
@@ -67,12 +67,12 @@ number, and followed by 3 bytes of color.
 
 ## Registers
 
-    rw brightness = 15: u8 / @ intensity
+    rw brightness: u0.8 / @ intensity
```

Comment:
```suggestion
    rw brightness = 0.05: u0.8 / @ intensity
```

Reply:
= 0.05 does not parse

Reply:
I see. My TODO keeps growing: 

## Spectool
* check if events are under events in spectool
* check if typical max etc are in range in spectool
* `u8 /` to `u0.8 /` in spectool (error)
* check if something follows undefined-sized type (eg: repeats: x: string) in spectool
* const register at 0x80 doesn't cause compile error
* disallow i0.X (only i1.X is ok)
* = 0.05 (led pixel bright) does not parse

I guess you did one though :)

Reply:
if you have a issue todo i can add notes :)